docs/user-guide/federation/configmap.md

```diff
@@ -0,0 +1,86 @@
+---
+---
+
+This guide explains how to use ConfigMaps in Federation control plane.
+
+* TOC
+{:toc}
+
+## Prerequisites
+
+This guide assumes that you have a running Kubernetes Cluster
+Federation installation. If not, then head over to the
+[federation admin guide](/docs/admin/federation/) to learn how to
+bring up a cluster federation (or have your cluster administrator do
+this for you). Other tutorials, for example
+[this one](https://github.com/kelseyhightower/kubernetes-cluster-federation)
+by Kelsey Hightower, are also available to help you.
+
+You are also expected to have a basic
+[working knowledge of Kubernetes](/docs/getting-started-guides/) in
+general and [ConfigMaps](/docs/user-guide/ConfigMaps/) in particular.
+
+## Overview
+
+ConfigMaps in federation control plane (referred to as "federated ConfigMaps" in
+this guide) are very similar to the traditional [Kubernetes
+ConfigMaps](/docs/user-guide/configmap/) providing the same functionality.
+Creating them in the federation control plane ensures that they are synchronized
+across all the clusters in federation.
+
+
+## Creating a Federated ConfigMap
+
+The API for Federated ConfigMap is 100% compatible with the
+API for traditional Kubernetes ConfigMap. You can create a ConfigMap by sending
+a request to the federation apiserver.
+
+You can do that using [kubectl](/docs/user-guide/kubectl/) by running:
+
+``` shell
+kubectl --context=federation-cluster create -f myconfigmap.yaml
+```
+
+The '--context=federation-cluster' flag tells kubectl to submit the
+request to the Federation apiserver instead of sending it to a kubernetes
+cluster.
+
+Once a federated ConfigMap is created, the federation control plane will create
+a matching ConfigMap in all underlying kubernetes clusters.
+You can verify this by checking each of the underlying clusters, for example:
+
+``` shell
+kubectl --context=gce-asia-east1a get configmap myconfigmap
+```
+
+The above assumes that you have a context named 'gce-asia-east1a'
+configured in your client for your cluster in that zone.
+
+These ConfigMaps in underlying clusters will match the federated ConfigMap.
+
+
+## Updating a Federated ConfigMap
+
+You can update a federated ConfigMap as you would update a Kubernetes
+ConfigMap; however, for a federated ConfigMap, you must send the request to
+the federation apiserver instead of sending it to a specific Kubernetes cluster.
+The Federation control plan ensures that whenever the federated ConfigMap is
```

Comment:
Also, different occurrences of Federation control plane use different cases. I think this should be made consistent. Lower case consistently everywhere is fine. Same for federation apiserver as well.